engine/flink/components/base/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/transformers.scala

```diff
@@ -112,28 +112,32 @@ object transformers {
                                sessionWindowTrigger: SessionWindowTrigger,
                                variableName: String
                               )(implicit nodeId: NodeId): ContextTransformation =
-    // TODO: to be consistent with sliding window we should probably forward context of variables for tumblingWindowTrigger == SessionWindowTrigger.OnEnd
-    ContextTransformation.definedBy(aggregator.toContextTransformation(variableName, emitContext = false, aggregateBy))
+  // TODO: to be consistent with sliding window we should probably forward context of variables for tumblingWindowTrigger == SessionWindowTrigger.OnEnd
```

Comment:
I think we can remove the comment safely :wink:

Reply:
removed